grass7/imagery/i.gabor/i.gabor.html

```diff
@@ -79,9 +79,9 @@ <h2>EXAMPLE</h2>
 </center>
 
 <div class="code"><pre>
-i.gabor input=ortho_2001_t792_1m output=ortho_i_8o_41_5_q \
+i.gabor input=ortho_2001_t792_1m output=ortho_i_8o_41_5 \
```

Comment:
Please make it completely reproducible including region settings.
```bash
g.region n= s= e= w= res=
```
It was the extent. I had to zoom into this specific area manually to be able to see the reproduced result.

![image](https://user-images.githubusercontent.com/7456117/121792222-39464d00-cbc0-11eb-8a7e-d34386dd19b0.png)